client/tcp_client.cpp

```diff
@@ -0,0 +1,63 @@
+#include "tcp_client.hpp"
+#include <future>
+#include <boost/asio/use_future.hpp>
+#include <array>
+#include <iostream>
+
+namespace fizyr
```

Comment:
I think its better to not use fizyr yet. Perhaps some other name, like `abc_client` would be better.

Reply:
I'll change

Reply:
Changed the namespace to async.